code/modules/surgery/organ_manipulation.dm

```diff
@@ -39,7 +39,6 @@
 	possible_locs = list(BODY_ZONE_CHEST, BODY_ZONE_HEAD)
 	requires_bodypart_type = BODYPART_ROBOTIC
 	lying_required = FALSE
-	self_operable = TRUE
```

Comment:
I am going to OK this ONLY if you disallow self-surgery on head and arms.

Reply:
Nah, but I changed it to take 3 times as long and always have a min failing chance of 5%. No reason to block head and arms. Merge merge merge

Reply:
> Nah, but I changed it to take 3 times as long and always have a min failing chance of 5%. No reason to block head and arms. Merge merge merge

I still want people to be dependent on other people to do surgery in some cases, how about head being disallowed when doing self surgery?

Reply:
self surgery should be gruelingly hard, give it a base 25% chance of failure minimum.